CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+The NuGet project tracks issues and feature requests using the [GitHub issue tracker](https://github.com/NuGet/Home/issues) for the various [NuGet repositories](https://github.com/NuGet/Home#repos-and-projects).
+
+## Before Submitting an Issue
+First, please do a search in [open issues](https://github.com/NuGet/Home/issues) to see if the issue or feature request has already been filed. Use this [query](https://github.com/NuGet/Home/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) to search for the most popular feature requests.
+
+If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment.
+
+👍 - upvote
+
+👎 - downvote
+
+The NuGet project is distributed across multiple repositories, try to file the issue against the correct repository [Related Projects](https://github.com/NuGet/Home#repos-and-projects).
+
+If your issue is a question then please ask the question on [Stack Overflow](https://stackoverflow.com/questions/tagged/nuget) using the tag `nuget`. Additionally, you can ask a question using [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/topics/dotnet-package-management.html) under the `package management` section.
```

Comment:
Is the `nuget` tag on Stack Overflow... good? Like is this an effective way for a struggling user to actually get help? I'm curious if we have some idea of what the user experience is in this regard. I looked at the [`nuget`](https://stackoverflow.com/questions/tagged/nuget) tag on StackOverflow and see mostly unanswered questions.

Reply:
It's a very low traffic tag, but it is one method to be used alongside the Microsoft Q&A, GitHub, etc. We should have more questions on SO imo!

![image](https://user-images.githubusercontent.com/1288848/104244186-8a8ebc80-5427-11eb-8957-03d864a07c36.png)

Reply:
contributing